routes/auth.routes.js

```diff
@@ -0,0 +1,32 @@
+const {Router} = require ('express')
+const router = new Router()
+const bcryptjs = require('bcryptjs')
+const saltRounds =10
+const User = require ('../models/User.model')
+
+// GET route ==> to display the signup form to users
+router.get ('/signup', (req,res) => res.render('views/signup.hbs'))
```

Comment:
the `views` section is always included; based on your file structure, if you wanted to render the signup page, you would just do `res.render('signup')`